manifests/init.pp

```diff
@@ -31,6 +31,7 @@
   },
   $createkeys_path     = '/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin',
   $manage_eyaml_config = true,
+  $manage_keys         = true,
```

Comment:
We need to validate our inputs and when dealing with booleans, handle the case of stringified booleans. This can be done with

``` puppet
if is_string($manage_keys) == true {
  $manage_keys_real = str2bool($manage_keys)
} else {
  $manage_keys_real = $manage_keys
}
validate_bool($manage_keys_real)
```